research/format.markdown

```diff
@@ -0,0 +1,138 @@
+## Draft 2 to Draft 3:
+
+> Behaviour: Not required to validate → MAY validate(but aren’t required to)
+
+**\-**  **street-address** : This should be a street address.
+
+**\-**  **locality** : This should be a city or town.
+
+**\-**  **region** : This should be a region (a state in the US, province in Canada, etc.)
```

Comment:
Also, below you are uppercasing "SHOULD" but in this section it is lowercase. Please be consistent!